lua/orgmode/config/defaults.lua

```diff
@@ -102,6 +102,7 @@ return {
       org_todo = 'cit',
       org_todo_prev = 'ciT',
       org_toggle_checkbox = '<C-Space>',
+	  org_toggle_heading = '<Leader>oh',
```

Comment:
_nit_: Perhaps `<Leader>o*` to match what _Emacs_ has for this binding. However we've strayed pretty far at this point so not sure it matters.

Reply:
+1 on this. `<Leader>o*` seems fine to me.